src/sentry/integrations/jira_server/utils/choice.py

```diff
@@ -0,0 +1,17 @@
+def build_user_choice(user_response):
```

Comment:
Same comment pretty much, I think it's worth still referencing `client.get_user_id_field` in some way so we're not repeating the string "name" in two locations that don't know about each other